utils/chunk-tests.js

```diff
@@ -3,10 +3,13 @@ const path = require('path');
 
 const NUMBER_OF_CHUNKS = 5;
 const MINIMUM_PER_CHUNK = 1;
-const runnersMap = new Map([['test-integration-once', ['ubuntu-latest']]]);
+const runnersMap = new Map([
+  ['test-integration-once', ['ubuntu-latest']],
+  ['test-integration-dev', ['ubuntu-latest', 'macos-latest']],
+]);
 
 async function getChunkedTests() {
-  const scripts = ['test-integration-once'];
+  const scripts = ['test-integration-once', 'test-integration-dev'];
```

Comment:
Should this be `runnersMap.keys()` instead of defining a new array?

Reply:
Updated in https://github.com/vercel/vercel/pull/7822/commits/f0ccb8115583ca0796335296406442a232babf5b